apps/partner2/stylesheets/overview.styl

```diff
@@ -28,7 +28,7 @@
       min-height 100px
       .partner-overview-section-title
         avant-garde-size('m-headline')
-    &[data-module=about] + [data-module=news]
+    &[data-module=news]
```

Comment:
So as per our research that determined that the previous check `&[data-module=about] + [data-module=news]` applied the style below to `.news` only in case those `.news` were coming as a sibling to the same parent directly after `.about` I think your change correctly applies the style below to news in all the cases.

I am wondering though what the original intention was here. Like why the original `.partner-overview-section-title` is hidden? Maybe @starsirius can have some memory here about the original design?